zulip_bots/zulip_bots/game_handler.py

```diff
@@ -165,10 +181,24 @@ def handle_message(self, message: Dict[str, Any], bot_handler: Any) -> None:
                 self.add_user_to_cache(message)
                 logging.info('Added {} to user cache'.format(sender))
 
+            if self.is_single_player:
+                if content.lower().startswith('start game with ') or content.lower() == 'accept' or \
+                        content.lower() == 'decline' or content.lower() == 'register' or content.lower() == 'draw' \
```

Comment:
We should be able to clean up this code by saying:

~~~
if content.lower().startswith('start game with ') or content.lower() in ['accept', 'decline', 'register', ...
~~~

But it would be better to try to do a deeper cleanup of this code to set up maps of command prefixes to functions, similar to what the virtual_fs bot does.